<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Begin</class>
 <widget class="QDialog" name="Begin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Start Game</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server host:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="SwitchControl" name="sw" native="true">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>11</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Server mod</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>10</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Socket mod</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="for_server">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="waiting">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>68</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#009688;&quot;&gt;Waiting...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="local_ip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Local IP:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Send request</string>
   </property>
  </widget>
  <zorder>for_server</zorder>
  <zorder>lineEdit</zorder>
  <zorder>ip</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>sw</zorder>
  <zorder>send</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchControl</class>
   <extends>QWidget</extends>
   <header location="global">switchcontrol.h</header>
   <container>1</container>
   <slots>
    <signal>toggled(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
